compiler/ballerina-lang/src/main/java/org/ballerinalang/model/tree/NodeKind.java

```diff
@@ -130,6 +130,7 @@
     TRANSACTIONAL_EXPRESSION,
     OBJECT_CTOR_EXPRESSION,
     ERROR_CONSTRUCTOR_EXPRESSION,
+    DYNAMIC_PARAM_EXPR,
```

Comment:
We need to rename this also.

```suggestion
    DYNAMIC_ARG_EXPR,
```